include/snowflake/BindUploader.hpp

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2024 Snowflake Computing, Inc. All rights reserved.
+ */
+
+#pragma once
+#ifndef SNOWFLAKECLIENT_BINDUPLOADER_HPP
+#define SNOWFLAKECLIENT_BINDUPLOADER_HPP
+
+#include <chrono>
+#include <snowflake/Exceptions.hpp>
+#include "client.h"
+
+namespace Snowflake
+{
+namespace Client
+{
+
+class BindUploader
```

Comment:
with add this file to include/snowflake for ODBC usage? if so we should consider it not as a public API of the driver - could we put some "internal" notice on top of this header file?

Reply:
and for 2.x we can mark in that way all files that we should consider internal

Reply:
we had some discussion with @sfc-gh-jszczerbinski about the headers and C++/C API - one of the possible future improvement is creating C++ API and having it expose additionally in C
it would be a topic for 3.x version probably